mne/viz/tests/test_ica.py

```diff
@@ -296,9 +296,23 @@ def test_plot_ica_scores():
     ica.labels_['eog'] = 0
     ica.labels_['ecg'] = 1
     ica.plot_scores([0.3, 0.2], axhline=[0.1, -0.1])
+    ica.plot_scores([[0.3, 0.2], [0.3, 0.2]], axhline=[0.1, -0.1])
     ica.plot_scores([0.3, 0.2], axhline=[0.1, -0.1], labels='foo')
     ica.plot_scores([0.3, 0.2], axhline=[0.1, -0.1], labels='eog')
-    ica.plot_scores([0.3, 0.2], axhline=[0.1, -0.1], labels='ecg')
+    # ica.plot_scores([0.3, 0.2], axhline=[0.1, -0.1], labels='ecg')
```

Comment:
cleanup

Reply:
That's an old test. I'm not sure what the point of it is nor what the desired/intended behavior is in that situation. see my comments on the PR.

I'm fine with deleting it just want to make sure I'm not changing some behavior that is used/usefull

Reply:
Can you change the ECG threshold so that it *does* find a component, and thus can plot? Alternatively, keep this line in but nested in a proper `pytest.raises`? Though I think the former would be preferable..

Reply:
I don't think there is any actual ECG thresholding going on.

The test manually sets the labels as : 
```
    ica.labels_ = dict()
    ica.labels_['eog/0/foo'] = 0
    ica.labels_['eog'] = 0
    ica.labels_['ecg'] = 1
```
but the plotter only looks for nested label names `label.startswith('ecg/')`

one solution would be to first look for nested labels ('ecg/') and if none are found look for non-nested ('ecg')

I don't like keeping the line and nesting in a `pytest.raises` that would convert a test that used to check if something didn't raise an error to a test that the same code now does raise and error.

Reply:
> one solution would be to first look for nested labels ('ecg/') and if none are found look for non-nested ('ecg')

I have no idea if the labeling is supposed to work this way, it would require digging into the code...

Maybe just add:
```
ica.labels_['ecg/0/bar'] = 1
```
? This should allow it to plot